packages/function-enhancers/src/debounce.ts

```diff
@@ -0,0 +1,75 @@
+export interface Options {
+  leading?: boolean;
+  trailing?: boolean;
+  maxWait?: number;
+}
+
+export default function debounce<F extends (...args: any[]) => void>(
```

Comment:
A TSDoc comment might be nice here to get some tooltip explanations of what the options here do. We do some of that in Quilt libraries such as `react-form` and I think here it could be helpful.